andrada12/assignment3/Dictionary.java

```diff
@@ -0,0 +1,62 @@
+
+import java.util.ArrayList;
+import java.util.HashMap;
+
+/*
+ * To change this license header, choose License Headers in Project Properties.
+ * To change this template file, choose Tools | Templates
+ * and open the template in the editor.
+ */
+
+/**
+ *
+ * @author andrada
+ */
+public class Dictionary {
+    
+    static HashMap<String, ArrayList<String>> dictionary = new HashMap<String, ArrayList<String>>();
```

Comment:
It would be nice to have either:
 - a comment
 - or a good variable name (better) 
so we can tell this is a map from prefixes to all words that start with that prefix.

I'd probably call to variable something like "prefixesToWords" but choose something that reads well for you.